mappings/net/minecraft/client/gui/widget/PagedEntryListWidget.mapping

```diff
@@ -81,3 +82,7 @@ CLASS net/minecraft/class_703 net/minecraft/client/gui/widget/PagedEntryListWidg
 			ARG 2 mouseX
 			ARG 3 mouseY
 			ARG 4 button
+	CLASS class_709 ListEntry
+	CLASS class_710 LabelSupplierEntry
+		FIELD field_2980 labelSupplier Lnet/minecraft/class_690$class_691;
```

Comment:
this is a bit conc

Reply:
what should it do
it supplies the label